types/react-native-datawedge-intents/index.d.ts

```diff
@@ -0,0 +1,21 @@
+// Type definitions for react-native-datawedge-intents 0.1
+// Project: https://github.com/darryncampbell/react-native-datawedge-intents#readme
+// Definitions by: DerZersaeger <https://github.com/DerZersaeger>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface ExtrasObject {
+    action: string;
+    extras?: object;
+}
+
+interface FilterObject {
+    filterActions: ReadonlyArray<string>;
+    filterCategories: ReadonlyArray<string>;
+}
+
+declare const DataWedgeIntents: {
+    sendBroadcastWithExtras: (extrasObject: ExtrasObject) => void;
+    registerBroadcastReceiver: (filter: FilterObject) => void;
+};
+
+export default DataWedgeIntents;
```

Comment:
It seems like the [underlying library](https://unpkg.com/react-native-datawedge-intents@0.1.2/index.js) is not using `export default`, so this .d.ts file should not either. Instead, you should follow the appropriate template from [the TypeScript handbook](http://www.typescriptlang.org/docs/handbook/declaration-files/templates.html). Note that if you want to use `import name from 'package';` in your consuming code, you can use the `esModuleInterop` compiler setting.